packages/react-core/src/operations/aggregation.js

```diff
@@ -0,0 +1,71 @@
+import { AggregationTypes } from './constants/AggregationTypes';
+
+const dummyFn = ([value]) => value;
+
+export const aggregationFunctions = {
+  [AggregationTypes.COUNT]: (values) => values.length,
+  [AggregationTypes.MIN]: min,
+  [AggregationTypes.MAX]: max,
+  [AggregationTypes.SUM]: sum,
+  [AggregationTypes.AVG]: avg
+};
+
+export function aggregate(feature, keys, operation) {
+  const normalizedKeys = Array.isArray(keys) ? keys : normalizeKeys(keys);
```

Comment:
Do we need the previous check with `Array.isArray(keys)` ?

Reply:
Nope, thanks 👌🏻